pkg/cloud-provider/securitygroup/securitygroup.go

```diff
@@ -15,15 +15,15 @@
 package securitygroup
 
 import (
+	cloud "antrea.io/nephe/apis/crd/v1alpha1"
```

Comment:
nit: Order the inputs into groups.

1. common imports

2. antrea imports

3. nephe imports

Reply:
Solved